lib/engine/round/g_1846/operating.rb

```diff
@@ -77,27 +77,32 @@ def can_place_token?
         private
 
         def ignore_action?(action)
-          return false if action.is_a?(Action::SellShares) && action.entity.corporation?
-
           case action
+          when Action::Bankrupt
```

Comment:
this must have not gotten merged? this file looks old

Reply:
Ah ok - was the merge only 24hrs ago? Wasn’t sure from the comments.

Reply:
Yeah the bankruptcy stuff didn't get merged, I've just started to take a look at getting it up to date with the recent step change. Soon, hopefully!